packages/common/src/ui/components/inputs/TextArea/TextArea.tsx

```diff
@@ -9,21 +9,33 @@ export const TextArea: FC<StandardTextFieldProps> = ({
   minRows = 4,
   InputProps,
   ...props
-}) => (
-  <BasicTextInput
-    sx={{ width: '100%' }}
-    InputProps={{
-      ...InputProps,
-      sx: {
-        backgroundColor: 'white',
-        ...InputProps?.sx,
-      },
-    }}
-    multiline
-    value={value}
-    onChange={onChange}
-    maxRows={maxRows}
-    minRows={minRows}
-    {...props}
-  />
-);
+}) => {
+  const [buffer, setBuffer] = React.useState(value);
```

Comment:
Is this now `BufferedTextArea`? I feel like it deserves to be renamed with these changes

Reply:
I've added a whole new one, though that's probably overkill